README.md

```diff
@@ -7,68 +7,132 @@ PADD (formerly Chronometer2) is a more expansive version of the original chronom
 ***Note:** PADD has been adopted by the Pi-hole team, thanks to JPMCK for creating this helpful tool!
 
 ## Setup PADD
-*More in-depth information about setting up PADD can be found in this repo’s [wiki](https://github.com/pi-hole/PADD/wiki/Setup).*
 
 - Get a copy of PADD by running:
-
-```bash
-cd ~
-wget -O padd.sh https://install.padd.sh
-```
-or
-```bash
-cd ~
-curl -sSL https://install.padd.sh -o padd.sh
-```
+  ```bash
+  cd ~
+  wget -O padd.sh https://install.padd.sh
+  ```
+  or
+  ```bash
+  cd ~
+  curl -sSL https://install.padd.sh -o padd.sh
+  ```
 
 - Make PADD executable by running
-
-```bash
-sudo chmod +x padd.sh
-```
-
-- Set PADD to auto run by adding the following to the end of `~/.bashrc`:
-
-```bash
-# Run PADD
-# If we’re on the PiTFT screen (ssh is xterm)
-if [ "$TERM" == "linux" ] ; then
-  while :
-  do
-    ./padd.sh
-    sleep 1
-  done
-fi
-```
-
-One line version
-
-```bash
-cd ~ ; echo "if [ \"\$TERM\" == \"linux\" ] ; then\n  while :\n  do\n    ./padd.sh\n    sleep 1\n  done\nfi" | tee ~/.bashrc -a
-```
-
-- Reboot your Pi-Hole by running `sudo reboot`. PADD should now run when your Pi-Hole has completed booting.
+  ```bash
+  sudo chmod +x padd.sh
+  ```
+
+## Using PADD
+
+- Run PADD from Pi-hole
+  ```bash
+  sudo ./padd.sh --server pi.hole
+  ```
+
+- Run PADD from other machine without 2FA
```

Comment:
I would rather make this a two step example:
One is for running it from a remote device. And the other would be an optional section where users are supplying their password and/or 2FA

Reply:
should it be removed all together since PADD will prompt for TOTP if it's needed

Reply:
I think it's fine to have one to illustrate that users have the ability to add those information as arguments.

Reply:
check latest commit. I moved the "running from other device" to below the docker example